src/docs/sphinx/user_guide/simple_conduction_tutorial.rst

```diff
@@ -0,0 +1,299 @@
+  to the input file.  This is because the configuration options are all part of the ``ThermalConduction::InputOptions``
+  extracted from Inlet.
+
+Configuring Material Conductivity
+---------------------------------
+
+Instead of using a monolithic material model, the ``ThermalConduction`` module currently allows for material parameters
+like conductivity, specific heat capacity, and density to be configured individually.
+
+Using C++
+"""""""""
+
+.. literalinclude:: ../../../../examples/simple_conduction/without_input_file.cpp
+   :start-after: _conductivity_start
+   :end-before: _conductivity_end
+   :language: C++
+
+Using Lua
+"""""""""
+
+.. literalinclude:: ../../../../examples/simple_conduction/conduction.lua
+   :start-after: _conductivity_start
+   :end-before: _conductivity_end
+   :language: Lua
+
+Setting Thermal (Dirichlet) Boundary Conditions
+-----------------------------------------------
+
+The following snippets add two Dirichlet boundary conditions:
+
+- One that constrains the temperature to 1.0 at boundary attribute 1, which for this mesh corresponds
+  to the side of the square mesh along the x-axis, i.e., the "bottom" of the mesh.
+- One that constrains the temperature to :math:`x^2 + y - 1` at boundary attributes 2 and 3, which for
+  this mesh correspond to the right side and top of the mesh, respectively.
+
+Using C++
+"""""""""
+
+.. literalinclude:: ../../../../examples/simple_conduction/without_input_file.cpp
+   :start-after: _bc_start
+   :end-before: _bc_end
+   :language: C++
+
+Using Lua
+"""""""""
+
+.. literalinclude:: ../../../../examples/simple_conduction/conduction.lua
+   :start-after: _bc_start
+   :end-before: _bc_end
+   :language: Lua
+
+.. note::
+  The exact names here are not critical, any entry whose name contains the string ``temperature``
+  will be applied as a Dirichlet condition to the temperature field.
+
+Setting Output Type
+-------------------
+
+.. hint::
+  Serac currently supports ``VisIt``, ``ParaView``, ``GLVis``, and ``SidreVisit`` output.
+
+Using C++
+"""""""""
+
+.. literalinclude:: ../../../../examples/simple_conduction/without_input_file.cpp
+   :start-after: _output_type_start
+   :end-before: _output_type_end
+   :language: C++
+
+Using Lua
+"""""""""
+
+In the input file (``conduction.lua``):
+
+.. literalinclude:: ../../../../examples/simple_conduction/conduction.lua
+   :start-after: _output_type_start
+   :end-before: _output_type_end
+   :language: Lua
+
+In the C++ driver:
+
+.. literalinclude:: ../../../../examples/simple_conduction/with_input_file.cpp
+   :start-after: _output_type_start
+   :end-before: _output_type_end
+   :language: C++
+
+Running the Simulation
+----------------------
+
+Now that we've configured the ``ThermalConduction`` instance using a few of its configuration
+options, we're ready to run the simulation.  We call ``completeSetup`` to "finalize" the simulation
```

Comment:
I would add a note that this allocate and builds all of the internal finite element data structures.